src/sections/Testimonies.tsx

```diff
@@ -20,7 +20,10 @@ interface CardProps {
 }
 
 const Card = ({ title, quote, img, entity }: CardProps) => (
-  <div className="flex flex-col bg-white p-7 text-left flex-1">
+  <div
+    tabIndex={0}
```

Comment:
nit: when navigating using tab, the pagination disappears.

Reply:
Yeah it's that way for everytime we have this behavior. Could be improved for sure but the important thing is we can access everything.